content/doc/book/getting-started/installing.adoc

```diff
@@ -0,0 +1,177 @@
+---
+layout: section
+---
+:doctitle: Installing Jenkins
+:notitle:
+:description:
+:author: Liam Newman
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+
+[NOTE]
+====
+This is still very much a work in progress
+====
+
+[IMPORTANT]
+====
+This section is part of _Getting Started_.
+It provides a golden path for a *basic* jenkins configuration on a number of platforms.
+It *DOES NOT* cover the full range of considerations or options for installing Jenkinks.
+See link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
+====
+
+== Overview
+
+
+== Pre-install
+
+=== System Requirements
+
+WARNING: These are *starting points*.  For a full discussion of factors see: link:/doc/book/hardware-recommendations/[Discussion of hardware recommendations]
+
+Minimum configuration:
```

Comment:
I suggest "Minimum recommended configuration" or something like that. You can run Jenkins in less than 512MB of RAM and definitely with less than 5GB of disk space.